jvm/hermes/src/main/jni/JJSIValue.cpp

```diff
@@ -39,8 +39,7 @@ void JJSIPreparedJavaScript::registerNatives() {
 }
```

Comment:
Can't put a comment on a diff that isn't modified - comment at the top of this file can now be removed as well:
https://github.com/player-ui/player/blob/dc3c3ab6fbc032b6dc44bdda553cfb5b616d93bf/jvm/hermes/src/main/jni/JJSIValue.cpp#L6-L15